hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2504,25 +2524,29 @@
       name: chip_sw_otp_ctrl_lc_signals
       desc: '''Verify the broadcast signals from LC ctrl.
 
-            - `lc_escalate_en_i`: read the error code CSR and verify that it reflects an FSM error
-              state due to the escalation signal triggering.
             - `lc_creator_seed_sw_rw_en_i`: verify that the secret2 partition is locked.
             - `lc_seed_hw_rd_en_i`: verify that the keymgr outputs a default value when enabled.
             - `lc_dft_en_i`: verify that the test interface within OTP ctrl is accessible.
             - `lc_check_byp_en_i`: verify that the background check during LC ctrl state
               programming passes when enabled.
 
+            Note that `lc_escalate_en_i` is verified via a connectivity test.
+
             X-ref'ed with chip_sw_lc_ctrl_broadcast test, which verifies the connectivity of the LC
             decoded outputs to other IPs.
             '''
       stage: V2
-      tests: []
+      tests: [
+        "chip_prim_tl_access",        // lc_dft_en_i
+        "chip_sw_lc_ctrl_transition", // lc_check_byp_en_i
+        // TODO: access control signals.
+        ]
     }
     {
       name: chip_sw_otp_ctrl_ast
-      desc: '''Verify the power sequencing signals to AST, as well as the alert.
+      desc: '''Verify the power sequencing signals to AST.
 
-            Details TBD.
+            TODO: make this a connectivity test (the functionality is not modelled in open-source).
```

Comment:
So once we add it to connectivity test, we can remove this testpoint, and add it to close source testplan right?

Reply:
I double-checked the connectivity test-plan: we already have checks for this in place.
I have hence removed this testpoint in this PR, and took note that this is something to add to the closed-source OTP testplan.